wowchemy/assets/css/reveal_custom.css

```diff
@@ -0,0 +1,16 @@
+// Mermaid.js diagram div
```

Comment:
This file is intended for the site admin and should be left empty. Why have you duplicated CSS code here from another file?

Reply:
That leaked from my own site, should be removed in next push